app/src/main/kotlin/com/wire/android/notification/MessageNotificationManager.kt

```diff
@@ -32,16 +32,18 @@ class MessageNotificationManager
     private val notificationManagerCompat: NotificationManagerCompat,
     private val notificationManager: NotificationManager
 ) {
+
     fun handleNotification(newNotifications: List<LocalNotificationConversation>, userId: QualifiedID, userName: String) {
         if (newNotifications.isEmpty()) return
 
         val activeNotifications: Array<StatusBarNotification> = notificationManager.activeNotifications ?: arrayOf()
 
-        showSummaryIfNeeded(userId, activeNotifications, userName)
         newNotifications.forEach {
             showConversationNotification(it.intoNotificationConversation(), userId, activeNotifications)
         }
 
+        showSummaryIfNeeded(userId, activeNotifications, userName)
```

Comment:
Why is the order changed? 🤔

Reply:
There was a problem with sound play order which caused that sometimes ping messages had normal notification sound. It was discussed with @borichellow